apt/minimization/minimizer.py

```diff
@@ -161,7 +168,7 @@ def fit(self, X=None, y=None):
         # take into account that estimator, X, y, cells, features may be None
 
         if X is not None and y is not None:
-            X, y = check_X_y(X, y, accept_sparse=True)
+            # X, y = check_X_y(X, y, accept_sparse=True)
```

Comment:
Is there somewhere we can move this check to (for example within an if that check it's a numpy array)?